src/theme/index.hbs

```diff
@@ -148,21 +148,18 @@
                         <i class="fa fa-search"></i>
                     </button>
                     {{/if}}
+                    <a href="{{ path_to_root }}print.html" title="Print this documentation" aria-label="Print this book"><i id="print-button" class="fa fa-print" style="padding-left: 5px;transform: translate(0px,1px) scale(0.8);"></i></a>
 
                 </div>
                 <div class="cust_column cust_right">
 
-                    <a title="" href="https://www.luos.io/#technology"><i class="fas fa-microchip"></i> Technology</a>
-
-                    <a title="Download on Github" target="_blank" rel="noopener noreferrer" href="https://github.com/luos-io"><i class="fab fa-github fa-lg" aria-hidden="true"></i> Download</a>
-
-                    <a title="Join the Luos Community" href="https://community.luos.io"><i class="fab fa-discourse fa-lg" aria-hidden="true"></i> Community</a>
-
-                    <a title="" href="https://www.luos.io/appstore"><i class="far fa-calendar-check"></i> App'Store</a>
-
-                    <a title="" href="https://www.luos.io/pricing"><i class="far fa-money-bill-alt fa-lg"></i> Pricing</a>
+                    <a title="" href="https://www.luos.io/#technology">Technology</a>
```

Comment:
The arrow in the website and forum navbar indicate that user will leave the website where he is.
So, here you should add an arrow in "technology", "Pricing", "Appstore", "Blog".
And open this link in a new tab I suggest to.